database/insertData/03_template_E_drugRegistration.js

```diff
@@ -932,6 +932,8 @@ exports.queries = [
               }
```

Comment:
Please remove lines 20-22 not needed anymore : )

Reply:
7dfb3ea28ca36d95152f8e8aa518d6aaa17c035b goneski